services/app-graphql/src/schema.graphql

```diff
@@ -904,6 +1105,7 @@ type Rate {
     stateCode: String!
     "Fuller state data for the submitting state"
     state: State!
+    "Number of contracts a state has"
```

Comment:
I forget if the frontend actually uses this. The stateNumber is used to generate the rateName. This isn't the number of contracts the state has but rather the unique number this rate has in the given state. it's funny how hard this is to describe! Every rate has a unique stateNumber in creation order that is part of rateName

Reply:
"number of contracts this rate relates to" or "number of contracts this rate is a subsidiary of" or "number of parent contracts for this rate", perhaps? I know what you mean about words being hard. we do not display this number anywhere.